symfony/ux-meta/1.0/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+    "bundles": {
+        "Symfony\\WebpackEncoreBundle\\WebpackEncoreBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "assets/": "assets/",
+        "config/": "%CONFIG_DIR%/",
+        "package.json": "package.json",
+        "webpack.config.js": "webpack.config.js"
+    },
+    "aliases": ["ux", "encore", "webpack", "webpack-encore"],
+    "gitignore": [
+        "/node_modules/",
+        "/%PUBLIC_DIR%/build/",
+        "npm-debug.log",
+        "yarn-error.log"
+    ],
+    "conflict": {
+        "symfony/framework-bundle": "<5.4"
```

Comment:
this conflict is probably not necessary anymore once you remove the webpack_encore.yaml file btw